src/main/scala/nl.knaw.dans.easy.license/Command.scala

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.license
+
+import java.io.FileOutputStream
+
+import nl.knaw.dans.lib.error._
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+import resource._
+
+import scala.language.reflectiveCalls
+import scala.util.control.NonFatal
+import scala.util.{ Failure, Success, Try }
+import internal._
+
+object Command extends App with DebugEnhancedLogging {
+  type FeedBackMessage = String
+
+  val configuration = Configuration()
+  val commandLine: CommandLineOptions = new CommandLineOptions(args, configuration) {
+    verify()
+  }
+  val app = new LicenseCreatorApp(configuration)
+  managed(app)
+    .acquireAndGet(app => {
+      for {
+        msg <- runSubcommand(app)
```

Comment:
no need for a for-comprehension if you do only one thing...
just do `.acquireAndGet(runSubcommand)`. that's enough